src/main.js

```diff
@@ -154,7 +154,7 @@ function showMakeYourOwnForm() {
 function showSavedPosters() {
   mainPosterSection.classList.add('hidden');
   savedPostersSection.classList.remove('hidden');
-  gridView();
+  createGridView();
```

Comment:
Awesome - thanks for checking / noticing that we forgot to use a verb in our function name!